packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -435,7 +452,7 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 
 		// Create a local ID for the blob. After uploading it to storage and before returning it, a local ID to
 		// storage ID mapping is created.
-		const localId = uuid();
+		const localId = this.uuidOverride ? this.uuidOverride() : uuid();
```

Comment:
consider just setting this.uuidOverride  to ()=>uuid() if its not passed, rather than dealing with undefined on every upload.

Reply:
might also rename to something like localBlobIdGenerator, as something that could be interesting in the future would be to try using the id compressor